lib/modules/manager/npm/extract/index.spec.ts

```diff
@@ -1027,6 +1027,87 @@ describe('modules/manager/npm/extract/index', () => {
         ],
       });
     });
+
+    it('extracts dependencies from pnpm.overrides, with version ranges in flat syntax', async () => {
+      const content = codeBlock`{
+        "pnpm": {
+          "overrides": {
+            "foo>bar": "2.0.0",
+            "foo@1.0.0": "2.0.0",
+            "foo@>1.0.0": "2.0.0",
+            "foo@>=1.0.0": "2.0.0",
+            "foo@1.0.0>bar": "2.0.0",
+            "foo@>1.0.0>bar": "2.0.0",
+            "foo@>=1.0.0 <2.0.0": ">=2.0.0"
+          }
+        }
+      }`;
```

Comment:
Let me know if you'd prefer for me to split these into `it.each` blocks or something more granular. The big blob of tests is not as nice for debugging, but Jest does quite well at displaying diffs. Happy to implement it either way 😌